examples/docs/en-US/input.md

```diff
@@ -682,7 +682,8 @@ Search data from server-side.
 |----| ----| ----|
 | blur | triggers when Input blurs | (event: Event) |
 | focus | triggers when Input focuses | (event: Event) |
-| change | triggers when the icon inside Input value change | (value: string \| number) |
+| change | triggers when the Input value changes and loses focus | (value: string \| number) |
+| input | triggers when the Input value changes | (value: string \| number) |
```

Comment:
Input event is implied by Vue. There is no need to include it in doc.

Reply:
Ah, I forgot it's part of `v-model` feature..do you think it's not worth any mention?

Other than that, description of `change` needs to be fixed:
>triggers when the icon inside Input value change
what icon? :D

Reply:
Element is a Vue framework, we expect user to be competent in Vue before using Element. The change's description indeed requires fixing. :D

Reply:
I am saying that Docs doesn't mention `v-model` either, so that needs to be assumed too :D 

Regarding `change` event: should we mention the difference between `input` and `change` (that it triggers after blurring), or just fix `icon` error?

Reply:
Yes. v-model is also assumed knowledge. It is covered extensively in vue's documentation.

Just fix the error.  `change` event generally behaves similarly to native component's change event.